qiskit_nature/second_q/transformers/active_space_transformer.py

```diff
@@ -189,68 +182,67 @@ def transform(self, problem: BaseProblem) -> BaseProblem:
                                electrons or orbitals are requested than are available, or if the
                                number of selected active orbital indices does not match
                                `num_molecular_orbitals`.
+            NotImplementedError: TODO.
         """
         if not isinstance(problem, ElectronicStructureProblem):
             raise QiskitNatureError(
                 "Only an ElectronicStructureProblem can be transformed by this Transformer, "
                 f"not problems of type, {type(problem)}."
             )
 
+        # TODO: re-implement non-MO case
+        if problem.basis != ElectronicBasis.MO:
+            raise NotImplementedError()
+            # TODO: suggest the user to fix this by first running a BasisTransformer
```

Comment:
I would like suggestions: do you think raising an error, requesting the user to apply a `BasisTransformer` suffices? Anything else would require additional input parameters to this transformer, which tell us the transformation coefficients, since these are no longer nested inside the problem